chips/e310_g003/src/interrupts.rs

```diff
@@ -0,0 +1,56 @@
+//! Named interrupts for the E310-G003 chip.
+
+#![allow(dead_code)]
+
```

Comment:
```suggestion
```

Reply:
Because not all interrupts are treated in `service_interrupt()` removing that line would result in useless warnings.

Reply:
Perhaps then make these `pub(crate)`? I removed the allow from g002 and didn't get any warnings.

Reply:
> 

I think you are referring deleting `pub` from `pub mod interrupts;`